appengine/standard/i18n/README.md

```diff
@@ -60,7 +60,7 @@ Move into this project directory and invoke the following command:
 
 This command creates a `locales/messages.pot` file in the `locales`
 directory which contains all the string found in your Python code and
-Jija2 tempaltes.
+Jija2 templates.
```

Comment:
Hey @timgates42 ! Thanks for this contribution! It looks like there's actually a second typo - would you be willing to add the correction for Jinja2 as well please? 
```suggestion
Jinja2 templates.
```